plugins/csr/fu-csr-device.c

```diff
@@ -397,7 +397,7 @@ fu_csr_device_probe (FuUsbDevice *device, GError **error)
 {
 	FuCsrDevice *self = FU_CSR_DEVICE (device);
 
-	/* devices have to be whitelisted */
+	/* devices have to be quirked */
```

Comment:
I think instead of quirked - `allowed`

Reply:
the comment is kinda obscure anyway, I've rewritten it to make sense.